app.js

```diff
@@ -4,13 +4,16 @@ const {
   getTopics,
   getAllEndpoints,
   getArticleById,
+  getArticles,
 } = require('./controlers/controlers');
 
 app.use(express.json());
```

Comment:
You don't need this line (express.json()) as it is only for endpoints that use the methods patch or post. Remove it until you need it :)